File: Programming-Basics-with-Python/9.While Loop - Lab/06. Max Number.py
# команда -> число под формата на текст или "Stop"
# повтарям: чета число + проверка дали е макс
# спирам: команда == "Stop"
# продължавам: команда != "Stop"
import sys
command = input()
max = -sys.maxsize
while command != "Stop":
    # command -> число под формата на текст
    number = int(command)
    # проверка за максимум
    if number > max:
        max = number

    command = input()
else: # command == "Stop"
    print(max)

File: Programming-Basics-with-JavaScript/5.While Loop/06maxNumber.js
function maxNumber(input){
    let maxNumber = -999999999
    let index = 0
    let command = input[index]
    index++
    while (command != 'Stop'){
        number = Number(command)
        if (maxNumber < number){
            maxNumber = number

        }
        command = input[index]
        index++


    }
    console.log(maxNumber)



}
maxNumber(["45",
"-20",
"7",
"99",
"Stop"])

